code/Test_Definitions/NumberVerification_verify.feature

```diff
@@ -0,0 +1,137 @@
+
+
+@NumberVerification_verify
+Feature: Camara Number Verification API verify
+
+# Input to be provided by the implementation to the tests
+# References to OAS spec schemas refer to schemas specified in
+# https://raw.githubusercontent.com/camaraproject/NumberVerification/main/code/API_definitions/number_verification.yaml
+
+  Background: Common Number Verification verify setup
+    Given the resource "/number-verification/v0"  as  base url
+    And the header "Content-Type" is set to "application/json"
+    And the header "Authorization" is set to a valid access token
+    And the header "x-correlator" is set to a UUID value
+    And the request body is compliant with the schema NumberVerificationRequestBody
+    And the response body is compliant with the schema NumberVerificationMatchResponse
+    And the header "x-correlator" is set to a UUID value
+    And NUMBERVERIFY_VERIFY_MATCH_PHONENUMBER1 is compliant with the schema DevicePhoneNumber
+    And NUMBERVERIFY_VERIFY_MATCH_PHONENUMBER2 is compliant with the schema DevicePhoneNumber
+    And NUMBERVERIFY_VERIFY_MATCH_PHONENUMBER1 is different to NUMBERVERIFY_VERIFY_MATCH_PHONENUMBER2
+
+  @NumberVerification_verify0_phoneNumber_does_not_match_schema
+  Scenario Outline: phoneNumber value does not comply with the schema
+    Given the request body property "$.phoneNumber" is set to: <phone_number_value>
+    When the HTTP "POST" request is sent
+    Then the response status code is 400
+    And the response property "$.status" is 400
+    And the response property "$.code" is "INVALID_ARGUMENT"
+    And the response property "$.message" contains a user friendly text
+
+    Examples:
+      | phone_number_value |
+      | string_value       |
+      | 1234567890         |
+      | +12334foo22222     |
+      | +00012230304913849 |
+      | 123                |
+      | ++49565456787      |
+
+  @NumberVerification_verify1_xcorrelator_does_not_match_schema
+  Scenario Outline: x-correlator request header value does not comply with the schema
+    Given if the optional request header "x-correlator" is set to: <xcorrelator_value>
+    When the HTTP "POST" request is sent
+    Then the response status code is 400
+    And the response property "$.status" is 400
+    And the response property "$.code" is "INVALID_ARGUMENT"
+    And the response property "$.message" contains a user friendly text
+
+    Examples:
+      | xcorrelator_value  |
+      | string_value       |
+      | boink              |
+
+
+  @NumberVerification_verify100_match_true
+  Scenario:  verify phone number NUMBERVERIFY_VERIFY_MATCH_PHONENUMBER1, network connection and access token matches NUMBERVERIFY_VERIFY_MATCH_PHONENUMBER1
+    Given they use the base url over a mobile connection
+    And the resource is "/verify"
+    And they acquired a valid access token associated with NUMBERVERIFY_VERIFY_MATCH_PHONENUMBER1 through OIDC authorization code flow or CIBA
```

Comment:
Note that for the other API test definition (like sim swap or device location) we did not mention this step. Perhaps the obtention of the access token should be considered as a prerequisite to the Test Definition?

Reply:
I would like to keep the acquisition of a valid access token in the tests.
Maybe in "Background"-section?

The tests cannot run without an access token and with NumberVerification it is important that the phoneNumber in the request matches the phoneNumber in the access token. 
I think that is true for all APIs. No access token, no test.

I would argue that the other feature files (sim swap and device location) should add access token acquisition.

Reply:
Adding this point in "Background"-section make a lot sense!
Works for me.
we need now to add it in all other test definition :)

Reply:
added access token acquisition to Background section